lib/console_web/controllers/router/device_controller.ex

```diff
@@ -163,6 +163,95 @@ defmodule ConsoleWeb.Router.DeviceController do
   end
 
   def add_device_event(conn, %{"device_id" => device_id} = event) do
+    if true do
```

Comment:
should we update this to use an env variable so it does not run w/ rabbitmq for undesired environments or open source?